Core/src/main/java/com/plotsquared/core/plot/flag/types/BlockTypeListFlag.java

```diff
@@ -91,4 +85,24 @@ protected BlockTypeListFlag(List<BlockTypeWrapper> blockTypeList, Caption descri
         return tabCompletions;
     }
 
+    private BlockTypeWrapper getCategory(final String blockString) throws FlagParseException {
+        if (!blockString.startsWith("#")) {
```

Comment:
Why did we need to move this inside getCategory? Obviously still works but I think it makes more send in the parent method.

Reply:
I thought it would be more readable